getting-started/ecosystem.md

```diff
@@ -0,0 +1,254 @@
+# Ecosystem
+
+WDL has a rich, distributed ecosystem of interconnected developer tools and execution
+engines to ensure (a) users can quickly write high-quality, idiomatic workflows, and (b)
+scaling the execution of those workflows within any computation environment is a breeze.
+
+::: tip
+
+The WDL ecosystem is rapidly evolving, and, while we're always looking to expand
+the list of known ecosystem tools, sometimes tools get missed. If you know of a tool
+that needs to be listed here but isn't, we encourage you to [create a pull request] and
+let us know!
+
+:::
+
+## Execution Engines
```

Comment:
@claymcleod this seems like too many places to keep track of. Can we link this to the main OpenWdl website and update: https://openwdl.org/wdl-dev/#execution-engines-and-platforms

Reply:
or @claymcleod  should we update the main website on Openwdl to link to this page? and replace this page https://openwdl.org/wdl-dev/#execution-engines-and-platforms?

Reply:
I proposed my solution here: https://github.com/openwdl/openwdl.github.io/pull/13.